src/Statistics/include/Statistics/BinomialConfidenceInterval.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <cstdint>
+
+namespace orbit_statistics {
+
+struct BinomialConfidenceInterval {
+  float lower;
+  float upper;
+};
+
+class BinomialConfidenceIntervalEstimator {
+ public:
+  [[nodiscard]] virtual BinomialConfidenceInterval Estimate(float ratio, uint32_t trials) const = 0;
+  virtual ~BinomialConfidenceIntervalEstimator() = default;
+};
+
+// Estimates 95% binomial confidence intervals using Wilson method.
+class WilsonBinomialConfidenceIntervalEstimator : public BinomialConfidenceIntervalEstimator {
+ public:
+  [[nodiscard]] BinomialConfidenceInterval Estimate(float ratio, uint32_t trials) const override;
+
+ private:
+  // (1 - 0.05/2)-quantile of the standard normal distribution.
+  constexpr static double kNormalQuantile = 1.959963985;
+  constexpr static double kNormalQuantile2 = kNormalQuantile * kNormalQuantile;
```

Comment:
Nit: Are these needed here? They could probably go into the cpp file, right?

Reply:
Do you suggest just to put these in `namespace orbit_statistics` in the .cpp file? But this way they will not be clearly related to `WilsonBinomialConfidenceIntervalEstimator`. Is that OK?

Reply:
You can actually, just put them as `static constexpr`s into the implementation of `Estimate`.

Reply:
Good point. Thanks!